src/components/login/LoginTypeSelector.jsx

```diff
@@ -18,14 +18,17 @@ const LoginTypeSelector = props => (
         <LoginType
             className={'fab fa-facebook'}
             loginTypeName={'Facebook'}
+            href={process.env.FACEBOOK_URL}
```

Comment:
#### Critical
Same as above. Extract the urls from the global env object first.